chains/scripts/add_users.js

```diff
@@ -0,0 +1,50 @@
+async function loadem(starting_page = 0) {
+    var cash_nonce = await api.query.cash.nonces({eth: saddle.account});
+    await feedAccount(cash_nonce, starting_page);
+}
+
+async function feedAccount(n, r, accounts = [] ) {
+    const user = saddle.account
+    var accounts
+    var cash_nonce = n
+    if (r > 2000) return;
+
+    if (accounts.length == 0 || i == accounts.length) {
+        console.log(r)
+        accounts =  
+            await fetch(`https://api.compound.finance/api/v2/account?page_number=${r}&page_size=200`)
```

Comment:
Wouldn't it just have been easier to do `0x000000000...1`, `0x000000000...2` and so on?

Reply:
yeah, maybe I'll change it up and do both so we get "realer" data and also more pathological data.